backend/utils.py

```diff
@@ -174,6 +175,25 @@ def offset_pagination(page, number_fetchs, query):
     return [query, more]
 
 
+def paginate(params, query):
```

Comment:
A better name is query_paginated. Add a well-explained docstring, renaming params to something more explicit and explain in the docstring what it receives. This function should receive page and limit of elements as parameters as optional.